.github/workflows/build.yaml

```diff
@@ -30,3 +30,7 @@ jobs:
 
         - name: Test
           run: bazel test //test-suite:all
+
+        - name: External Test
+          working-directory: external-test
+          run: bazel run @djinni//src:djinni -- --help
```

Comment:
Adding a simple test here to ensure we can consume the compiler from an external workspace. This seems trivial but it will fail if we can't build and/or run